README.md

```diff
@@ -3,18 +3,47 @@ A list of resources for data science work on climate change
 
 ## Data Sets
 
+### Global
+* [ICE Embodied Carbon Database](https://circularecology.com/embodied-carbon-footprint-database.html)
+* [Carbon in food downloadable csv](https://ourworldindata.org/food-choice-vs-eating-local)
+* [EPW Weather files (for Ladybug on Grasshopper, but adaptable)](https://www.ladybug.tools/epwmap/)
+* [Renewable Ninja - front end for NASA dataset](https://www.renewables.ninja/)
+* [One of many NASA datasets. This one covers precipitation](https://disc.gsfc.nasa.gov/datasets/GPM_3IMERGHH_06/summary?keywords=IMERG)
+
+* [ Keen cyclist route heatmap](https://www.strava.com/heatmap)
+* [Havard atlas of economic complexity downloads](https://atlas.cid.harvard.edu/data-downloads)
+* [Open Street map API](https://operations.osmfoundation.org/policies/api/)
+
+
+### National
```

Comment:
Lets not make this separation right now between `National` and `City`, I want to organise this by tool, not by scope of applciation.

Reply:
The UK datasets are pretty worthless for an American.  The Bogota ai data is amazing, but it's City level, not national or global. For the rain storage project, I deliberately wanted to use a global dataset and not national.
The scope is important.

Reply:
This is subdiviidng the Data Set tool list by scope. Scope matters.